### PR TITLE
Add UoExeter CVs

### LINKED ISSUE #76: Add CI step which checks that web pages are up to date before merging
Pretty simple. Right now, it is in theory possible to update the database and not update the associated web pages. We should add CI to make sure that the web pages are up to date. This could have the same pattern as the 'check database is up to date' CI.

### PR DESCRIPTION
**Pull request by znichollscr**
## Description

Closes #76 (this update was a good excuse to check the functionality)

## Checklist

Please confirm that this pull request has done the following:

- [ ] Documentation added (where applicable) (N/A)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

**Comment by znichollscr** [comment]
@thomasaubry can you double check and confirm this is correct please

File: CVs/input4MIPs_source_id.json [hunk 1]
```diff
@@ -89,5 +89,13 @@
...
         "source_version":"4.3"
+    },
+    "UOEXETER-CMIP-0-1-0":{
+        "contact":"t.aubry@exeter.ac.uk",
+        "further_info_url":"http://www.tbd.invalid",
+        "institution_id":"uoexeter",
+        "license_id":"CC BY 4.0",
+        "mip_era":"CMIP6Plus",
+        "source_version":"0.1.0"
     }
```
Comment (znichollscr) [inline CVs/input4MIPs_source_id.json:100]: @thomasaubry just triple checking I copied this across correctly?

Comment (thomasaubry) [reply CVs/input4MIPs_source_id.json:100]: @znichollscr should it be 1-1-0 and 1.1.0? I've been pulling hair for a couple hours trying to upload my emission file on the ftp and wondering if it's related. It's now consistently 1-1-0 in my notebook and source_id file.

Comment (znicholls) [reply CVs/input4MIPs_source_id.json:100]: Perhaps. It's totally up to you. The change below is needed to be consistent with your notebook right?

```suggestion
    "UOEXETER-CMIP-1-1-0":{
        "contact":"t.aubry@exeter.ac.uk",
        "further_info_url":"http://www.tbd.invalid",
        "institution_id":"uoexeter",
        "license_id":"CC BY 4.0",
        "mip_era":"CMIP6Plus",
        "source_version":"1.1.0"
    }
```

Comment (znichollscr) [reply CVs/input4MIPs_source_id.json:100]: Perhaps. It's totally up to you. The change below is needed to be consistent with your notebook right?

```suggestion
    "UOEXETER-CMIP-1-1-0":{
        "contact":"t.aubry@exeter.ac.uk",
        "further_info_url":"http://www.tbd.invalid",
        "institution_id":"uoexeter",
        "license_id":"CC BY 4.0",
        "mip_era":"CMIP6Plus",
        "source_version":"1.1.0"
    }
```

Comment (znichollscr) [reply CVs/input4MIPs_source_id.json:100]: @thomasaubry ping here

**Pull request merged**